site/content/docs/private-locations/_index.md

```diff
@@ -11,7 +11,12 @@ aliases:
 cli: true
 ---
 
-A Private Location is an additional location in Checkly. It acts exactly like any other location, but you run it on your own infrastructure. A Private Location can be used to run checks on your services which are not directly exposed to the public internet but in your own segregated network. To operate a Private Location you need to run at least one Checkly Agent.
+A Private Location is an additional location in Checkly. It acts exactly like any other location, but you run it on your own infrastructure. To operate a Private Location you need to run at least one Checkly Agent.
+
+Private Locations allow you to run checks from within your own infrastructure, enabling testing and monitoring of both internal services (within your private network) and external services (publicly accessible).
+
+1. Internal Monitoring: Test the availability and performance of services that aren't accessible externally.
```

Comment:
@checklyalex this numbered list's connection to the previous paragraph is not obvious. Do we need a colon someplace?